cmd/helm/init.go

```diff
@@ -42,10 +47,21 @@ const (
 type initOptions struct {
 	skipRefresh         bool   // --skip-refresh
 	stableRepositoryURL string // --stable-repo-url
+	pluginsFilename     string // --plugins
 
 	home helmpath.Home
 }
 
+type pluginsFileEntry struct {
+	Name    string `json:"name,omitempty"`
```

Comment:
it doesn't look like `Name` is used anywhere. Should this field be removed and just rely on the URL and Version?

Reply:
I've removed it from the code and additionally documented the fact that the `name` field serves no functional purpose in `docs/plugins.md`.